std/algorithm/iteration.d

```diff
@@ -885,15 +893,18 @@ template each(alias pred = "a")
         (!isForwardRange!R || isDynamicArray!R) &&
         (isForeachUnaryIterable!R || isForeachBinaryIterable!R);
 
-    void each(Range)(Range r)
+    Flag!"each" each(Range)(Range r)
```

Comment:
They don't appear in the docs so choosing one or another is a wash. I preserved the existing structure.